<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveDataDock</class>
 <widget class="QWidget" name="LiveDataDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2" colspan="3">
    <widget class="QComboBox" name="cbUpdateType">
     <item>
      <property name="text">
       <string>Time interval</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New data</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" rowspan="2" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lSampleRate">
     <property name="text">
      <string>Sample rate: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lKeepNvalues">
     <property name="text">
      <string>Keep N values</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lUpdateType">
     <property name="text">
      <string>Update type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lUpdateFrequency">
     <property name="text">
      <string>Update frequency:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QPushButton" name="bUpdateNow">
     <property name="text">
      <string>Update now</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="QPushButton" name="bStopReading">
     <property name="text">
      <string>Stop reading</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QPushButton" name="bPausePlayReading">
     <property name="text">
      <string>Pause reading</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QSpinBox" name="sbUpdateFrequency">
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>3600000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLineEdit" name="leKeepNValues"/>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QSpinBox" name="sbSampleRate">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
